### ISSUE TITLE
[PHP] Ensure timezone is set in cli.php

### ISSUE THREAD

**Issue by naxhh** [phansible/phansible#198]
When testing #196 I get this error:

```
TASK: [php | ensure timezone is set in cli php.ini] ***************************
fatal: [192.168.33.99] => One or more undefined variables: 'timezone' is undefined

FATAL: all hosts have already failed -- aborting
```

Seems not related with the issue. The playbook was the default one + redis from the master branch. (Not 1.0 branch)

Needs confirmation.

**Comment by InFog** [comment]
So, I guess this one can be closed, right? I mean, I looks like #199 fixed it :)

**Comment by naxhh** [comment]
Right!
